meson.build

```diff
@@ -0,0 +1,102 @@
+project(
+  'pgroonga',
+  'c',
+  license: 'PostgreSQL',
+  version: '4.0.2')
+
+required_groonga_version = '14.0.0'
+
+groonga = dependency('groonga', version: '>= ' + required_groonga_version)
+cc = meson.get_compiler('c')
+libm = cc.find_library('m')
+
+pg_config = find_program('pg_config')
+pg_includedir_server = run_command(
+  pg_config, '--includedir-server', check: true
+).stdout().strip()
+pg_pkglibdir = run_command(
+  pg_config, '--pkglibdir', check: true
+).stdout().strip()
+pg_includedir_extension = pg_includedir_server / 'extension'
+
+pgrn_version = meson.project_version()
+pgrn_c_args = [f'-DPGRN_VERSION="@pgrn_version@"']
+if get_option('buildtype').contains('debug')
+  pgrn_c_args += ['-DPGROONGA_DEBUG=1']
+endif
+
+postgresql = declare_dependency(
+  compile_args:
+    run_command(pg_config, '--cflags_sl', check: true).stdout().strip().split(),
```

Comment:
Does Meson's `split()` work like shell? For example, is `a "b c" d` split to `["a", "b c", "d"]`?
If it's not compatible with shell, we should use it here.

Reply:
fix: [9a66ad1](https://github.com/pgroonga/pgroonga/pull/743/commits/9a66ad129a88eaa7b169d553665fee76e56cc721) The `split()` isn't compatible with shell. So I removed it here.